### DISCUSSION TITLE
alignment1

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#17] [Announcements]
# alignment1

Exploring the alignment problem in the context of AGI and societal challenges.

https://nonzerosum.games/alignment1.html

**Comment by FreeFallAnnie** [comment]
Thank you for writing about this! Here I offer those thoughts I promised.

Here's my optimistic view following my principle **'Life as Fractal'**.

These are the variables I'm considering, and doing so as such.
 - governments (currently the ones who can make decisions in regards to policy) operate in response to the people 
 - the people respond to their intake of their individual worlds
 - the individuals world is constructed by their memories, their values, their identity
 - which are hijacked by the companies they interact with (*cough targeting advertising)
 - the companies operate for their values which often go profit regardless of the detrimental effects - including pitting people against one another.

Okay so I'll address how I'm flowing this through my brainweb.

To me, fractals are a way of attaining alignment. (Our love 'recursion' rears their pretty head).

**How could we do so for humanity?**

As <a href=https://en.wikipedia.org/wiki/Stephen_Covey>Stephen Covey</a> put - start with your circle of influence.
1. What can WE as individuals control?
- we can start with our values and our own alignment

A challenge comes when the individual is influence by the companies.
2. But, what really is a company?
...if not a bunch of individuals aligning collectively?
Like the notions of <a href=https://www.ted.com/talks/seth_godin_the_tribes_we_lead>Tribes</a> - as put by Seth Godin -
*"A tribe is any group of people, large or small, who are connected to one another, a leader, and an idea."*

Now we have both individuals and companies collectives aligned.

I'm unsure of the answer, but is innovation & progress often actually dictated by private industry?

So comes, what about the government?
Yeh. About that.
It really is a good question.


(I'm not antigovernment, but I wonder about their role - to reflect the people. In my post's version of reality, government is a form of collective... and I don't know about them.... Would they reflect the people in face of change?)

*Random jumble of thoughts....* 
Making use of modular design of collectives - individuals could opt in and out of collectives, and it could operate with an intersystemic design, cybernetics - direction setting rather than destination stops - faith in humans as fundamentally a self-organising system *fingers crossed...

If we build up the fractals of the world beginning with us, could that influence the systems enough that - while the collectives have different objectives - we all contribute to the concept of humanity? When the manmade system is large enough it takes on ecosystem properties with viruses are able to be managed?

Curious for a feedback loop!
:D

**Reply by jamesstephenbrown** [reply]
Hey Annie, sorry about the slow iteration on my part in this feedback loop, been trying to write the ever growing number of posts in this series (looks like 6!).

You definitely have some really interesting ideas here. I love fractals (yes, probably something about emergence through recursion) and those sorts of dynamics definitely need to be factored into any process - strongly top down solutions are not going to work for this reason. I think government can definitely have a role, but mainly by augmenting incentives so that the fractal grows in a sustainable way - hopefully allowing for an infinite game, rather than a finite or boom-crash / peace-war cycle. Emergent systems can find some equilibrium, but it's not guaranteed to be one that aligns with human well-being.

I hope government can curb the natural tendency you highlight of companies to measure success by profit at the expense of externalities like the environment and inequality. I see government as a form of conscious choice within an emergent market system.

I haven't read "Tribes" yet, but am a fan of Godin, as a philosopher of the market essentially, but with a moral compass - unlike his counterpart in Peter Thiel.

I write later in the series about our natural tendencies towards forming collectives, and highlight to fact that this is an evolutionary trait that was developed to suit small bands of kin, and doesn't have the necessary feedback mechanisms in a global economy.

I like your idea about divergent collectives working in harmony with different objectives. I don't think this is incompatible with working towards alignment in general, the value of diversity in general has been shown on many levels, evolutionary, societal, cultural to have its advantages, so any adequately aligned picture of humanity would need to accomodate a good degree of diversity. That's one wrinkle in the alignment problem itself is the idea that the same goals can actually lead to conflict, if for instance there is a competition for the same resources, whereas having different motivations can lead to a harmonious situation where those interests can be pursued independently of one another without causing any conflict. Perhaps this is best described with a quadrant of; same interests in conflict (zero-sum), same interests in harmony and cooperation (non-zero), different interests in conflict (zero-sum), different interests in harmony (non-zero). There are two options that lead to alignment and two that lead to misalignment.

Hence why I see alignment and non-zero-sum games as strongly "aligned".

**Comment by jamesstephenbrown** [comment]
Hey Annie, sorry about the slow iteration on my part in this feedback loop, been trying to write the ever growing number of posts in this series (looks like 6!).

You definitely have some really interesting ideas here. I love fractals (yes, probably something about emergence through recursion) and those sorts of dynamics definitely need to be factored into any process - strongly top down solutions are not going to work for this reason. I think government can definitely have a role, but mainly by augmenting incentives so that the fractal grows in a sustainable way - hopefully allowing for an infinite game, rather than a finite or boom-crash / peace-war cycle. Emergent systems can find some equilibrium, but it's not guaranteed to be one that aligns with human well-being.

I hope government can curb the natural tendency you highlight of companies to measure success by profit at the expense of externalities like the environment and inequality. I see government as a form of conscious choice within an emergent market system.

I haven't read "Tribes" yet, but am a fan of Godin, as a philosopher of the market essentially, but with a moral compass - unlike his counterpart in Peter Thiel.

I write later in the series about our natural tendencies towards forming collectives, and highlight to fact that this is an evolutionary trait that was developed to suit small bands of kin, and doesn't have the necessary feedback mechanisms  in a global economy.

I like your idea about divergent collectives working in harmony with different objectives. I don't think this compatible with working towards alignment in general, the value of diversity in general has been shown on many levels, evolutionary, societal, cultural to have its advantages, so any adequately aligned picture of humanity would need to accomodate a good degree of diversity. That's one wrinkle in the alignment problem itself is the idea that the same goals can actually lead to conflict, if for instance there is a competition for the same resources, whereas having different motivations can lead to a harmonious situation where those interests can be pursued independently of one another without causing any conflict. Perhaps this is best described with a quadrant of; same interests in conflict (zero-sum), same interests in harmony and cooperation (non-zero), different interests in conflict (zero-sum), different interests in harmony (non-zero). There are two options that lead to alignment and two that lead to misalignment.

Hence why I see alignment and non-zero-sum games as strongly "aligned".

**Comment by jamesstephenbrown** [comment]
^ Oops, I changed direction mid-sentence in paragraph 6... I meant to say "I don't think this is incompatible with working towards alignment in general"

**Comment by andrewtaneglen** [comment]
RE: "...a key stumbling block in designing AGI that is aligned with humanity's interests is the fact that we cant agree on what humanity's interests are."

Aristotle's ethics are in large part an attempt to answer this question, but posed more as "what should we be doing", in the sense of finding our purpose or function, or in another way: we can easily understand and identify a flourishing plant, or ecosystem, so what would constitute a flourishing human being? What ends are the flourishing human being concerned with? In this sense, what are the interests of a flourishing human being?

This becomes quickly tied up with "what are the interests of a particular human being", these being the instrumental ends to take a particular human being to whatever this flourishing state might be. But they important to keep apart, as we should first come to agree on what the flourishing human life looks like before getting bogged down in the practicalities of achieving that.

This is reflected in Maslow's hierarchy of needs - in the sense that the flourishing state resembles self-actualisation - a state attained when all other practical needs are met. Similarly, without keeping self-actualisation in mind, we might become convinced the world is a zero-sum game and become tied up protecting and attaining ever greater quantities of our more basic instrumental needs. A billionaire on a superyacht is necessarily at the point of self-actualisation in this sense. They just have an absurd level of basic needs. Perhaps, in this sense, the fear of AI developing counterproductive instrumental goals is already exemplified by these billionaires with their instrumental goals gone-wild.

**Reply by jamesstephenbrown** [reply]
We are prone to projection - perhaps it's only us we need to worry about getting carried away with instrumental goals.

Even though Sam Harris has gone down some rabbit holes I'm no longer following him down, I did like his use of terms like "human flourishing" - which perhaps he gleaned from Aristotle - it really helps shift the perspective that individual and collective goals are at odds - it makes it obvious *everyone wants to live in a place where there is human flourishing*.

**Reply by andrewtaneglen** [reply]
Yeah, it's a funny word. Flourishing isn't quite right, or doesn't quite capture it without recalibrating your use of the term. Aristotle/Ancient Greek has the term 'Eudaimonia', generally taken (e.g. by Hursthouse) as "...a combination of well-being, happiness and flourishing. It is the ultimate goal of a human being...".

PS: that was meant to say "A billionaire on a superyacht is not necessarily at the point of self-actualisation in this sense."

**Reply by jamesstephenbrown** [reply]
Ah, re PS:, that makes more sense now :)
